.github/workflows/main.yml

```diff
@@ -158,6 +158,9 @@ jobs:
       - name: 'Switch to Xcode 14.1'
         run: sudo xcode-select -switch /Applications/Xcode_14.1.app
 
+      - name: 'aarch64-apple-darwin'
+        run: rustup target add aarch64-apple-darwin
```

Comment:
Just for less Rust-aware people like myself: This is only necessary for cross-compilation, right?